plyer/platforms/linux/inappbrowser.py

```diff
@@ -0,0 +1,14 @@
+
+from plyer.facades.inappbrowser import InAppBrowser
+import webbrowser
+
+
+class LinuxAppBrowser():
+
+    def _open_url(self, **kwargs):
+        url = kwargs.get('url', '')
```

Comment:
Yes it works on linux,windows and osx as stated on [official site](https://docs.python.org/2/library/webbrowser.html) I would love to do it but at present i lack resources to test ios and osx part.So here what i assumed will do in next commit.
- replace ' ' by kivy official site url.
- put the same present linux code under windows and osx.
If there is something more to edit plz suggest :)